3-comp/3-3-comp/components/AddProduct.vue

```diff
@@ -33,7 +33,7 @@
             return;
           }
           //4/ But instead of adding element we're just emitting onAddProduct event with product as a payload
-          this.$emit('onAddProduct', {
+          this.$emit('add-product', {
```

Comment:
Dlaczego z myślnikami? Dodatkowo komentarz jest u góry, który w takim przypadku też trzeba zmienić.

Reply:
styleguide zaleca camel-case do eventów

Reply:
https://vuejs.org/v2/guide/components-custom-events.html#Event-Names

Reply:
Tak, ale tutaj bardziej chodziło mi o to, żeby pokazać, że można. W sumie zarówno eventy jak i nazwy komponentów mogą być i tak i tak.

Reply:
Tak, ale eventy są z natury tylko w templacie używane dlatego tak zalecają. Komponenty zalecają pascalcase zeby łatwiej bylo importy robic i od znaczników natywnych i libek odróżniać. 

Zmienić to na wcześniejszą wersję czy zostawić tak jak zrobiłem?

Reply:
Tak, ale jak ktoś korzysta np. z Reacta to dla niego zmiany kebab-case na camelCase są nienaturalne ;) Możemy zmienić. Warto o tym pamiętać po prostu i wspomnieć na szkoleniu.